include/functionLog_tcpdf.php

```diff
@@ -0,0 +1,49 @@
+<?php
+// create 11:16 AM 10/17/2011
+function tsaLog($str, $type = 'i', $nl=true) { // 11:16 AM 10/17/2011
```

Comment:
I think this file and function should be removed from this pull-request

Reply:
yes it for debug only